package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "discord-tauri",
   "scripts": {
-    "tauri": "tauri"
```

Comment:
Thank you for making this, I have one question tough, won't removing "tauri" from the scripts make "yarn tauri" or "npm run tauri" not work?

Reply:
I mentioned `yarn serve` and `npm run server` in my comment. Would you like me to replace `serve` with `tauri`?